lib/tasks/ee.rake

```diff
@@ -0,0 +1,263 @@
+require 'yaml'
+require 'erb'
+
+class Database
+  RECOGNIZED_OPTIONS = %w(user host port socket)
+
+  def initialize(options = {})
```

Comment:
Ok, this is Ruby 2.1 only -- I'm not sure what Ruby version FEDs are using to run these tasks, so I'm going to skip for now.   Thanx guise.